data-explorer/kusto/query/best-practices.md

```diff
@@ -32,6 +32,7 @@ Here are several best practices to follow to make your query run faster.
 |  | Use `in` instead of left semi `join` for filtering by a single column. |  |
 | Join across clusters | Across clusters, run the query on the "right" side of the join, where most of the data is located. |  |
 | Join when left side is small and right side is large | Use [hint.strategy=broadcast](./broadcastjoin.md) |  | Small refers to up to 100,000 records. |
+| Join when right side is small and left side is large | Consider using [lookup operator](./lookupoperator.md) instead of `join` operator |  | Small refers to serveral tens of MBs. |
```

Comment:
```suggestion
| Join when right side is small and left side is large | Use the [lookup operator](./lookupoperator.md) instead of `join` |  | If the right side of the lookup is larger than several tens of MBs, the query will fail. |
```